frontend/components/instruments/PadInstrument.js

```diff
@@ -39,16 +67,66 @@ Pad.propTypes = {
  */
 const PadInstrument = ({samples}) => {
     const audioContextRef = useRef(new AudioContext());
+    const keyBinds = ['q', 'w', 'e', 'r', 'a', 's', 'd', 'f', 'u', 'i', 'o', 'p'];
+    const pads = [];
+    const pads2 = [];
+    const pads3 = [];
+    // eventually, would it be possible to make this a single for loop instead of 3? just for
+    // readability?
+    for (let i = 0; i < 4; i++) {
+        let padClassName = STYLES.cyanPad;
+        const thisPad = (
+            <Pad
+                keyBind={keyBinds[i]}
+                key={i}
+                sample={samples[i]}
+                padClassName={padClassName}
+                audioContext={audioContextRef.current}
+            />
+        );
+        pads.push(thisPad);
+    };
+    for (let i = 4; i < 8; i++) {
+        let padClassName = STYLES.magentaPad;
+        const thisPad = (
+            <Pad
+                keyBind={keyBinds[i]}
+                key={i}
+                sample={samples[i]}
+                padClassName={padClassName}
+                audioContext={audioContextRef.current}
+            />
+        );
+        pads2.push(thisPad);
+    };
+    for (let i = 8; i < 10; i++) {
+        let padClassName = STYLES.limePad;
+        const thisPad = (
+            <Pad
+                keyBind={keyBinds[i]}
+                key={i}
+                sample={samples[i]}
+                padClassName={padClassName}
+                audioContext={audioContextRef.current}
+            />
+        );
+        pads3.push(thisPad);
+    };
 
     return (
-        <div id="step-sequencer">
-            {samples.map((sample, i) => (
-                <Pad
-                    key={i}
-                    sample={sample}
-                    audioContext={audioContextRef.current}
-                />
-            ))}
+        // TODO: add a flexbox div so that padline1 + 3 can appear on the same line
```

Comment:
Is this what you are trying to get at here? (Pads colored for clarity)
<img width="373" alt="Screen Shot 2021-12-02 at 11 01 02 AM" src="https://user-images.githubusercontent.com/1494102/144457802-4c7b9bd1-1774-4156-ae02-0b828f389c26.png">